ipwb/util.py

```diff
@@ -144,6 +150,10 @@ def setIPWBReplayIndexPath(cdxj):
 
 def getIPWBReplayIndexPath():
     ipfsJSON = readIPFSConfig()
```

Comment:
This line is redundant and unnecessary as it is being set again after ensuring `setIPWBReplayConfig`.